ZuluConverter.py

```diff
@@ -21,3 +21,80 @@ def convert_cst_to_zulu(cst_datetime):
 print(convert_cst_to_zulu(datetime.today()))
 
 
+#EST to Zulu
+def convert_est_to_zulu(est_datetime):
+    # Define time zones
+    est_tz = timezone('America/New_York')    # Eastern Standard Time
+    zulu_tz = timezone('UTC')   # Universal Time (Zulu)
+
+    # Localize the input EST datetime
+    localized_est = est_tz.localize(est_datetime)
+
+    # Convert the localized EST datetime to Zulu time
+    zulu_datetime = localized_est.astimezone(zulu_tz)
+
+    zulu_datetime_formatted = zulu_datetime.strftime("%Y-%m-%dT%H:%M:%S.%fZ")
+
+    return zulu_datetime_formatted
+
+print(convert_est_to_zulu(datetime.today()))
+
+#PST to Zulu
+def convert_pst_to_zulu(pst_datetime):

```

Comment:
each of these functions has a lot, a lot, of duplication ... what about something like:
```python
def convert_pst_to_zulu(pst_datetime):
    convert_to_zulu(pst_datetime, 'America/Los_Angeles')

def convert_to_zulu(some_datetime, some_timezone_name):
    localized_datetime = timezone(some_timezone_name).localize(some_datetime)
    zulu_datetime = localized_datetime.astimezone(timezone('UTC'))
    formatted = zulu_datetime.strftime('%Y-%m-%dT%H:%M:%S.%fZ')
    return formatted
```